index.js

```diff
@@ -0,0 +1,94 @@
+
+console.log("Hello");
+function playRound(playerSelection_, computerSelection_) {
```

Comment:
pls remove the trailing underscore, Js variables should  not contain underscores